R/pkg/tests/fulltests/test_sparkSQL_arrow.R

```diff
@@ -19,7 +19,9 @@ library(testthat)
 
 context("SparkSQL Arrow optimization")
 
-sparkSession <- sparkR.session(master = sparkRTestMaster, enableHiveSupport = FALSE)
+sparkR.session(master = sparkRTestMaster,
+               enableHiveSupport = FALSE,
+               sparkConfig = list(spark.sql.execution.arrow.sparkr.enabled = "true"))
```

Comment:
Since we have all arrow tests in this file, setting it enabled by default makes more sense.